diff --git a/node_modules/@emotion/cache/dist/emotion-cache.cjs.dev.js b/node_modules/@emotion/cache/dist/emotion-cache.cjs.dev.js
index af9ac5a..195c3ff 100644
--- a/node_modules/@emotion/cache/dist/emotion-cache.cjs.dev.js
+++ b/node_modules/@emotion/cache/dist/emotion-cache.cjs.dev.js
@@ -159,9 +159,10 @@ var createUnsafeSelectorsAlarm = function createUnsafeSelectorsAlarm(cache) {
         return;
       }
 
-      unsafePseudoClasses.forEach(function (unsafePseudoClass) {
-        console.error("The pseudo class \"" + unsafePseudoClass + "\" is potentially unsafe when doing server-side rendering. Try changing it to \"" + unsafePseudoClass.split('-child')[0] + "-of-type\".");
-      });
+      // we're not doing SSR and not interested in pedantic errors
+      // unsafePseudoClasses.forEach(function (unsafePseudoClass) {
+      //   console.error("The pseudo class \"" + unsafePseudoClass + "\" is potentially unsafe when doing server-side rendering. Try changing it to \"" + unsafePseudoClass.split('-child')[0] + "-of-type\".");
+      // });
     }
   };
 };
diff --git a/node_modules/@emotion/cache/dist/emotion-cache.esm.js b/node_modules/@emotion/cache/dist/emotion-cache.esm.js
index fd65cd3..40899c2 100644
--- a/node_modules/@emotion/cache/dist/emotion-cache.esm.js
+++ b/node_modules/@emotion/cache/dist/emotion-cache.esm.js
@@ -150,9 +150,10 @@ var createUnsafeSelectorsAlarm = function createUnsafeSelectorsAlarm(cache) {
         return;
       }
 
-      unsafePseudoClasses.forEach(function (unsafePseudoClass) {
-        console.error("The pseudo class \"" + unsafePseudoClass + "\" is potentially unsafe when doing server-side rendering. Try changing it to \"" + unsafePseudoClass.split('-child')[0] + "-of-type\".");
-      });
+      // we're not doing SSR and not interested in pedantic errors
+      // unsafePseudoClasses.forEach(function (unsafePseudoClass) {
+      //   console.error("The pseudo class \"" + unsafePseudoClass + "\" is potentially unsafe when doing server-side rendering. Try changing it to \"" + unsafePseudoClass.split('-child')[0] + "-of-type\".");
+      // });
     }
   };
 };
