diff --git a/node_modules/adbkit-logcat/lib/logcat/parser/binary.js b/node_modules/adbkit-logcat/lib/logcat/parser/binary.js
index 12fa8ec..e122933 100644
--- a/node_modules/adbkit-logcat/lib/logcat/parser/binary.js
+++ b/node_modules/adbkit-logcat/lib/logcat/parser/binary.js
@@ -10,7 +10,7 @@ const HEADER_SIZE_MAX = 100
 class Binary extends EventEmitter {
   constructor(options) {
     super(options)
-    this.buffer = new Buffer(0)
+    this.buffer = Buffer.alloc(0)
   }
 
   parse(chunk) {
