diff --git a/node_modules/antd/es/button/button.js b/node_modules/antd/es/button/button.js
index c96d188..06a814b 100644
--- a/node_modules/antd/es/button/button.js
+++ b/node_modules/antd/es/button/button.js
@@ -189,6 +189,8 @@ var InternalButton = function InternalButton(props, ref) {
   }, [loadingOrDelay]);
   React.useEffect(fixTwoCNChar, [buttonRef]);
 
+  var scope = React.useContext(global.FlipperTrackingScopeContext);
+
   var handleClick = function handleClick(e) {
     var _a;
 
@@ -200,6 +202,13 @@ var InternalButton = function InternalButton(props, ref) {
       return;
     }
 
+    if(onClick !== null && onClick !== void 0) {
+      global.flipperTrackInteraction(
+        'Button', 'onClick', scope, props.title || props.children || props.icon,
+        onClick,
+        e
+      );
+     }
     (_a = onClick) === null || _a === void 0 ? void 0 : _a(e);
   };
 
