diff --git a/node_modules/antd/es/button/button.js b/node_modules/antd/es/button/button.js
index 3dce9d4..d046b84 100644
--- a/node_modules/antd/es/button/button.js
+++ b/node_modules/antd/es/button/button.js
@@ -184,6 +184,8 @@ var InternalButton = function InternalButton(props, ref) {
   }, [loadingOrDelay]);
   React.useEffect(fixTwoCNChar, [buttonRef]);
 
+  var scope = React.useContext(global.FlipperTrackingScopeContext);
+
   var handleClick = function handleClick(e) {
     var onClick = props.onClick;
 
@@ -192,7 +194,11 @@ var InternalButton = function InternalButton(props, ref) {
     }
 
     if (onClick) {
-      onClick(e);
+      global.flipperTrackInteraction(
+        'Button', 'onClick', scope, props.title || props.children || props.icon, 
+        onClick,
+        e
+      );
     }
   };
 
