diff --git a/node_modules/metro/src/lib/getPreludeCode.js b/node_modules/metro/src/lib/getPreludeCode.js
index 476319e..c1cd920 100644
--- a/node_modules/metro/src/lib/getPreludeCode.js
+++ b/node_modules/metro/src/lib/getPreludeCode.js
@@ -57,7 +57,7 @@ function getPreludeCode(_ref) {
     "__BUNDLE_START_TIME__=this.nativePerformanceNow?nativePerformanceNow():Date.now()",
     `__DEV__=${String(isDev)}`
   ].concat(_toConsumableArray(formatExtraVars(extraVars)), [
-    "process=this.process||{}",
+    "process=process||this.process||global.process||{}",
     `__METRO_GLOBAL_PREFIX__='${globalPrefix}'`
   ]);
   return `var ${vars.join(",")};${processEnv(
