diff --git a/node_modules/metro/src/lib/getPreludeCode.js b/node_modules/metro/src/lib/getPreludeCode.js
index f73c6ec..fc313ec 100644
--- a/node_modules/metro/src/lib/getPreludeCode.js
+++ b/node_modules/metro/src/lib/getPreludeCode.js
@@ -14,7 +14,7 @@ function getPreludeCode({ extraVars, isDev, globalPrefix }) {
     "__BUNDLE_START_TIME__=this.nativePerformanceNow?nativePerformanceNow():Date.now()",
     `__DEV__=${String(isDev)}`,
     ...formatExtraVars(extraVars),
-    "process=this.process||{}",
+    "process=process||this.process||global.process||{}",
     `__METRO_GLOBAL_PREFIX__='${globalPrefix}'`
   ];
   return `var ${vars.join(",")};${processEnv(
