diff --git a/node_modules/metro/src/index.js b/node_modules/metro/src/index.js
index 1c3a58e..96c7376 100644
--- a/node_modules/metro/src/index.js
+++ b/node_modules/metro/src/index.js
@@ -115,8 +115,6 @@ const createConnectMiddleware = async function (config, options) {
           wss.handleUpgrade(request, socket, head, (ws) => {
             wss.emit("connection", ws, request);
           });
-        } else {
-          socket.destroy();
         }
       });
     },
diff --git a/node_modules/metro/src/lib/getPreludeCode.js b/node_modules/metro/src/lib/getPreludeCode.js
index 56780d7..7da26d1 100644
--- a/node_modules/metro/src/lib/getPreludeCode.js
+++ b/node_modules/metro/src/lib/getPreludeCode.js
@@ -14,7 +14,7 @@ function getPreludeCode({ extraVars, isDev, globalPrefix }) {
     "__BUNDLE_START_TIME__=this.nativePerformanceNow?nativePerformanceNow():Date.now()",
     `__DEV__=${String(isDev)}`,
     ...formatExtraVars(extraVars),
-    "process=this.process||{}",
+    "process=process||this.process||global.process||{}",
     `__METRO_GLOBAL_PREFIX__='${globalPrefix}'`,
   ];
   return `var ${vars.join(",")};${processEnv(
